tyk-headless/configs/tyk_mgmt.conf

```diff
@@ -12,6 +10,7 @@
         "tags": []
     },
     "app_path": "/mnt/tyk-gateway/apps",
+    "enable_uptime_analytics": true,
```

Comment:
Why are we enabling this?

Reply:
Thank you for spotting this! 

I enabled this based on the instructions described [here](https://tyk.io/docs/tyk-pump/tyk-pump-configuration/tyk-pump-dashboard-config/#uptime-tests-analytics). However, since the headless does not come with Dashboard, we do not need to enable this field. 

I will update the PR accordingly 👍

Reply:
Updated @komalsukhani 🙏